contracts/market/IMarket.sol

```diff
@@ -79,8 +77,6 @@ interface IMarket {
 
     function collateralToken() external view returns (IERC20);
 
-    function paymentToken() external view returns (IERC20);
```

Comment:
Why remove from the interface?

Reply:
because it's not used in the IMarket interface